alpheios_nemo_ui/data/assets/scss/references.scss

```diff
@@ -1,112 +1,79 @@
-                color: $alpheioscolordark;
-                
-                
-            }
-        }
-
-        .browse-level-2,
-        .browse-level-3 {
-            display: none;
-        }
-    }
-    
-    .reference-browse-cards {
-        
-        @include collection-list-style();
-        margin-top: 0;
-
-        .list-item-link {
-            @include alpheios-link();
-            @include alpheios-neutral-dark();
-        }
-
-        .reference-browse-cards {
-            display: none; 
-            margin: 0;
-        }
-
-        .collection-browse-cards {
-            @include collection-cards-style();
-
-            li.small-card {
-                @include media-breakpoint-up(sm) {
-                    max-width: 200px;
-                }
-                @include media-breakpoint-up(md) {
-                    width: 200px;
-                }
-            }
-
-            li {
-                .left-part {
-                    .card-title {
-                        margin-bottom: 0;
-                    }
-                }
-                .right-part {
-                    max-width: 50px;
-                    .card-icon,
-                    .card-text {
-                        display: inline-block;
-                        margin-left: 0;
-                    }
-                    .card-icon {
-                        margin-bottom: 0;
-                    }
-                }
-            }
-
-            @include media-breakpoint-up(sm) {
-                li {
-                    .right-part {
-                        max-width: none;
-                    }
-                    .right-part .card-text {
-                        margin-left: 10px;
-                    }
-
-                    .left-part {
-                        .card-title {
-                            line-height: 1;
-                        }
-                    }
-            
-                    .right-part {
-                        .card-text {
-                            max-width: none;
-                        }
-                    }
-                }
-            }
-        }
-    }
-}
-.ref-book-item {
-    .passages {
-        display: none;
-    }
+.reference-browse-container {
+    @include collection-container();
+
+    .browse-header {
+        .book-level-prev {
+            @include alpheios-h3();
+            @include alpheios-vivid();
+            .link-icon {
+                @include alpheios-icon(16px, 16px);
+                svg {
+                    vertical-align: baseline;
+                }
+                @include media-breakpoint-up(md) {
+                    @include alpheios-icon(20px, 20px);
+                }
+            }
+            &:hover {
+                text-decoration: none;
+            }
+        }
+
+        .browse-level-1 {
+            .book-level-prev {
+                cursor: default;
+                color: $alpheioscolordark;
+                
+                

```

Comment:
A very minor thing, but it's probably better to remove empty lines here

Reply:
Fixed :)